.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
+env:
+- COVERAGE=
```

Comment:
ok, I'd rather do one run of the test suite with coverage enabled. If, at some point, we get suspicious of the results then we can start running the suite twice. The reason I'd like only one run of the test suite is to keep the tests very fast. I know they are fast now but they will only get slower over time. Also, the Flow startup is still a bit slow (3 seconds).